apps/fail2ban.sh

```diff
@@ -27,24 +27,23 @@ debug_mode
 root_check
 
 # Check if fail2ban is already installed
-if ! [ -f /etc/fail2ban/filter.d/nextcloud.conf ]
+if ! [ -f /etc/fail2ban/filter.d/nextcloud.conf ] || is_this_installed fail2ban
```

Comment:
The correct check should be
`if ! [ -f /etc/fail2ban/filter.d/nextcloud.conf ] && ! is_this_installed fail2ban`